roofit/roofitcore/src/RooAbsPdf.cxx

```diff
@@ -1047,20 +1047,37 @@ RooAbsReal* RooAbsPdf::createNLL(RooAbsData& data, const RooLinkedList& cmdList)
     // Simple case: default range, or single restricted range
     //cout<<"FK: Data test 1: "<<data.sumEntries()<<endl;
 
-    auto theNLL = new RooNLLVar(baseName.c_str(),"-log(likelihood)",
-        *this,data,projDeps,ext,rangeName,addCoefRangeName,numcpu,interl,
-        verbose,splitr,cloneData,/*binnedL=*/false, pc.getDouble("IntegrateBins"));
-    theNLL->batchMode(pc.getInt("BatchMode"));
+    RooAbsTestStatistic::Configuration cfg;
+    cfg.rangeName = rangeName;
+    cfg.addCoefRangeName = addCoefRangeName;
+    cfg.nCPU = numcpu;
+    cfg.interleave = interl;
+    cfg.verbose = verbose;
+    cfg.splitCutRange = static_cast<bool>(splitr);
+    cfg.cloneInputData = static_cast<bool>(cloneData);
+    cfg.integrateOverBinsPrecision = pc.getDouble("IntegrateBins");
+    cfg.binnedL = false;
+    cfg.batchMode = pc.getInt("BatchMode");
+    auto theNLL = new RooNLLVar(baseName.c_str(),"-log(likelihood)",*this,data,projDeps,std::move(cfg));
```

Comment:
The `ext` argument was missing in the last commit which caused the tests to fail.